stl/inc/algorithm

```diff
@@ -386,6 +386,21 @@ namespace ranges {
     inline constexpr _For_each_n_fn for_each_n{_Not_quite_object::_Construct_tag{}};
 
     // VARIABLE ranges::find
+    // clang-format off
+    template <input_iterator _It, sentinel_for<_It> _Se, class _Ty, class _Pj = identity>
+        requires indirect_binary_predicate<ranges::equal_to, projected<_It, _Pj>, const _Ty*>
+    _NODISCARD constexpr _It _Find_unchecked(_It _First, const _Se _Last, const _Ty& _Val, _Pj _Proj = {}) {
+

```

Comment:
Gah, sorry - I failed to remove the extra line from my suggestion here yesterday (Github magically adds an extra line to suggestions on files with CRLF line endings).
```suggestion
```